Source/p_spec.c

```diff
@@ -2064,6 +2067,19 @@ void P_PlayerInSpecialSector (player_t *player)
           // Tally player in secret sector, clear secret special
           player->secretcount++;
           sector->special = 0;
+
+          if (showMessages && hud_secret_message && player == &players[consoleplayer])
+          {
+            int sfx_id;
```

Comment:
Could we probably make this static and initialize it only once?

Reply:
`static int sfx_id;` ? I don't get it.

Reply:
`static int sfx_id = ...`. Static variables are only initialized once and then remembered across subsequent function calls. Once we have checked for the sound lumps, we know which one is available and which isn't. No need to check this every time this function is called.

Reply:
In  C static variables only initialized by constant.

Reply:
Ah, sure, there must have been a reason why I did it like this in Crispy. 😉